test/SecTester.Scan.Tests/CI/VendorMatcherTests.cs

```diff
@@ -0,0 +1,252 @@
+using System.Collections.Specialized;
+
+namespace SecTester.Scan.Tests.CI;
+
+public class VendorMatcherTests
+{
+  private readonly JsonSerializerOptions _options = new()
+  {
+    PropertyNameCaseInsensitive = true, PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+  };
+
+  public static readonly IEnumerable<object[]> MatchEnvInput = new List<object[]>
+  {
+    new object[]
+    {
+      @"{
+        ""env"": { ""env"": ""NODE"", ""includes"": ""/app/.heroku/node/bin/node""}
+        }",
```

Comment:
Avoid the wrong indentation
```suggestion
      @"{""env"": { ""env"": ""NODE"", ""includes"": ""/app/.heroku/node/bin/node""}}",
```